[2025-03-07T06:07:02.149+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-07T06:07:02.158+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_payments.etl_process manual__2025-03-07T06:06:58.959898+00:00 [queued]>
[2025-03-07T06:07:02.164+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_payments.etl_process manual__2025-03-07T06:06:58.959898+00:00 [queued]>
[2025-03-07T06:07:02.165+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-03-07T06:07:02.177+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_process> on 2025-03-07 06:06:58.959898+00:00
[2025-03-07T06:07:02.192+0000] {standard_task_runner.py:72} INFO - Started process 329 to run task
[2025-03-07T06:07:02.197+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_payments', 'etl_process', 'manual__2025-03-07T06:06:58.959898+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/etl-payments.py', '--cfg-path', '/tmp/tmpljdpwpkq']
[2025-03-07T06:07:02.200+0000] {standard_task_runner.py:105} INFO - Job 146: Subtask etl_process
[2025-03-07T06:07:02.254+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_payments.etl_process manual__2025-03-07T06:06:58.959898+00:00 [running]> on host 53355976fa1b
[2025-03-07T06:07:02.296+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_payments' AIRFLOW_CTX_TASK_ID='etl_process' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T06:06:58.959898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-07T06:06:58.959898+00:00'
[2025-03-07T06:07:02.297+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-07T06:07:02.314+0000] {etl-payments.py:31} INFO - ðŸ”¹ Mengambil data dari database sumber...
[2025-03-07T06:07:08.905+0000] {logging_mixin.py:190} WARNING - /opt/airflow/dags/etl-payments.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-03-07T06:07:14.781+0000] {etl-payments.py:36} INFO - âœ… Data berhasil diambil!
[2025-03-07T06:07:14.784+0000] {etl-payments.py:38} INFO - ðŸ”¹ Memproses data...
[2025-03-07T06:07:15.115+0000] {etl-payments.py:59} INFO - âœ… Data berhasil dibersihkan!
[2025-03-07T06:07:40.134+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-07T06:07:40.149+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-07T06:07:40.149+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_payments, task_id=etl_process, run_id=manual__2025-03-07T06:06:58.959898+00:00, execution_date=20250307T060658, start_date=20250307T060702, end_date=20250307T060740
[2025-03-07T06:07:40.212+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-07T06:07:40.230+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-07T06:07:40.230+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
